operations/patch.go

```diff
@@ -62,7 +63,11 @@ func Patch() cli.Command {
 				Large:       c.Bool(largeFlagName),
 				Alias:       c.String(patchAliasFlagName),
 				Ref:         c.String(refFlagName),
-				WorkingTree: c.Bool(workingChangesFlag),
+				Uncommitted: c.Bool(uncommittedChangesFlag),
+			}
+
+			if !params.Uncommitted {
+				grip.Infof("Uncommitted changes are omitted from patches by default.\nUse the '--%s' flag to include uncommitted changes.", uncommittedChangesFlag)
```

Comment:
I think this could be smarter, though I'm not sure how you feel about the increase in complexity: Could we only warn people who actually have uncommitted changes, not everyone?

Reply:
done.